specificatie/openapi.yaml

```diff
@@ -1,173 +1,152 @@
 openapi: 3.0.0
 servers:
-  - description: "SwaggerHub API Auto Mocking"
-    url: https://virtserver.swaggerhub.com/VNGRealisatie/api/reisdocumenten
+  - description: |
+                  APILAB testserver
+    url: https://www.haalcentraal.nl/haalcentraal/api/brp
 info:
   title: Reisdocumenten
-  description: "API voor het ontsluiten van gegevens van reisdocumenten uit de GBA en RNI."
-  version: "1.0.0"
+  description: |
+                API voor het ontsluiten van gegevens van reisdocumenten uit de GBA en RNI.
+  version: "0.0.1 (develop)"
   contact:
-    url: https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen
+    url: https://github.com/VNG-Realisatie/Haal-Centraal-Reisdocumenten-bevragen
   license:
     name: European Union Public License, version 1.2 (EUPL-1.2)
     url: https://eupl.eu/1.2/nl/
+tags:
+  - name: Reisdocumenten
 paths:
   /reisdocumenten/{reisdocumentnummer}:
     get:
-      operationId: reisdocumentenReisdocumentnummer
-      description: "Het ophalen van de gegevens van een REISDOCUMENT op basis van het reisdocumentnummer."
+      operationId: GetReisdocument
+      summary: Raadpleeg een reisdocument
+      description: |
+                    Raadpleeg een Reisdocument.
+                    Gebruik de fields parameter als je alleen specifieke velden in het antwoord wil zien, [zie functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-Reisdocumenten-bevragen/blob/v1.0.0/features/fields_extensie.feature).
+
       parameters:
         - in: path
           name: reisdocumentnummer
-          description: "Het nummer van het verstrekte Nederlandse reisdocument."
           required: true
           schema:
             type: string
             maxLength: 9
+        - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/parameters/fields"
```

Comment:
waarom is eigenlijk fields toegevoegd? Was hier specifiek vraag naar? We hebben toch geen afspraak dat een API altijd fields moet ondersteunen?

Bijvoorbeeld /ingeschrevenpersonen/{burgerservicenummer}/kinderen/{id} heeft ook geen fields parameter.

Reply:
Ik vond het een goed idee vanuit consistentie gedachte. Was inderdaad geen expliciete vraag naar. Ik heb hem weer verwijderd.